crates/subspace-service/src/lib.rs

```diff
@@ -516,3 +593,28 @@ pub async fn new_full(config: Configuration) -> Result<NewFull<Arc<FullClient>>,
         backend,
     })
 }
+
+/// Builds a new object suitable for chain operations.
+pub fn new_chain_ops(
```

Comment:
Why is this necessary? Doesn't seem to save much in terms of code.

Reply:
It's simplified from the original version https://github.com/paritytech/polkadot/blob/64059a2fab9878ffde4971db0c3dc15bcb4282f4/node/service/src/lib.rs#L1191-L1258 , but still save a few LOC comparing with using`new_partial` directly in `command.rs`. The advantage could be more obvious with multiple runtimes.